### DISCUSSION TITLE
Is Isaaclab suitable for satellite simulation?

### DISCUSSION THREAD

**Discussion by dxyy1** [isaac-sim/IsaacLab#1459] [Q&A] [answered]
Hi everyone, 

I was wondering if Isaaclab suitable for developing control algorithm for satellites?
In particular, I wonder if isaaclab:
1. can simulate variation in gravity?
2. has light sensors?
3. can detect variation in light intensity?

Thanks in advance :)
![image](https://github.com/user-attachments/assets/d54c5a3f-e12a-4cbe-aec3-2af7413c22fe)

**Comment by RandomOakForest** [comment]
This is a great question for the [Isaac Sim forums](https://forums.developer.nvidia.com/c/omniverse/simulation/69). Isaac Lab leverages Isaac Sim for simulation. Hope this helps.

**Reply by SirensAI** [reply, marked as answer]
I would say yes, You can define gravity in the properties in your case you would want to adjust the gravity modifier to know gravitational effects at the orbit(s) your satellite would be working at.  I would ask in the rendering section with regard to light, which I take it you want to model the amount of solar energy hitting your panels?

Richard3D would be the guy to tag in the forum for the rendering of the light hitting the surface. due to the ray tracing technology I think its a perfect fit
